warehouse/templates/search/results.html

```diff
@@ -23,7 +23,7 @@
     <h3 class="package-snippet__title">
       <span class="package-snippet__name">{{ item.name }}</span>
       <span class="package-snippet__version">{{ item.latest_version }}</span>
-      {% if "created" in order %}
+      {% if "created" in order or order == "" %}
```

Comment:
I think if we're showing this for "Relevance" and "Date last updated" we can probably just show it for "Trending" too and remove this conditional entirely.

Reply:
Works for me 7464024

![image](https://user-images.githubusercontent.com/3238190/85814330-8cf32880-b71a-11ea-88b6-32d964a922f7.png)

Reply:
Unfortunately I think you'll need to `make translations` now that we've removed lines.